### PR TITLE
Products updates (UTC time, local time string conversion, new columns, etc)

### PR DESCRIPTION
**Pull request by leonardolaiolo**
Remember to rebase this branch to master after https://github.com/aodn/public-schema/pull/34 is merged.
Then make sure `combined_bgc_map.sql` still includes the new filters as well UTC time.

### CODE REVIEW

File: bgc_data/combined_bgc_map.sql [hunk 1]
```diff
@@ -5,8 +5,8 @@ CREATE MATERIALIZED VIEW combined_bgc_map AS
     "Project",
     "StationName",
     "TripCode",
-    "SampleTime_local"::date AS "SampleDate_Local",
---  TODO: "SampleTime_UTC",
+    "SampleTime_local"::date::text AS "SampleDate_Local",
```
Comment (mhidas) [inline bgc_data/combined_bgc_map.sql:8]: For date I don't think we need to cast to `text` as Geoserver shouldn't add a `Z` to make it look like it's UTC (but we should check this in the stack).

**Review by mhidas — CHANGES_REQUESTED** [review]
Getting there!

File: bgc_data/bgc_phytoplankton_abundance_genus_data.sql [hunk 2]
```diff
@@ -25,7 +25,7 @@ WITH grouped AS (
     SELECT m.trip_code,
            g.genus,
            CASE
-               WHEN m."SampleTime_local" < g.startdate THEN NULL
+               WHEN m."SampleTime_local"::date < g.startdate THEN NULL
```
[the remark this answers is not in the document]
Comment (leonardolaiolo) [reply bgc_data/bgc_phytoplankton_abundance_genus_data.sql:28]: fixed in last commit

File: bgc_data/bgc_trip_metadata.sql [hunk 3]
```diff
@@ -7,15 +7,17 @@ CREATE VIEW bgc_trip_metadata AS
     latitude AS "Latitude",
     longitude AS "Longitude",
     trip_code AS "TripCode",
-  --  TODO: sampledatelocal AT TIME ZONE "UTC" AS "SampleTime_UTC",
-    sampledatelocal AS "SampleTime_local",
+    sampledateutc AT TIME ZONE 'UTC' AS "SampleTime_UTC",
```
[the remark this answers is not in the document]
Comment (leonardolaiolo) [reply bgc_data/bgc_trip_metadata.sql:10]: fixed in last commit

File: bgc_data/combined_bgc_map.sql (hunk 1 above)
Comment (leonardolaiolo) [reply bgc_data/combined_bgc_map.sql:8]: Correct, fixed in last commit - I double checked this in my stack and there's no added `Z` to either `SampleTime_local` or `SampleTime_UTC`

**Review by mhidas — APPROVED** [review]

**Pull request merged**